pkg/bpf/bpf.go

```diff
@@ -267,7 +267,7 @@ func ObjGet(pathname string) (int, error) {
 	)
 
 	if fd == 0 || err != 0 {
-		return 0, fmt.Errorf("Unable to get object: %s", err)
+		return 0, fmt.Errorf("Unable to get object: %s", pathname)
```

Comment:
I think we want to print both path and error:
`return 0, fmt.Errorf("Unable to get object %s: %s", pathname, err)`